### ISSUE TITLE
Multi-select with Selection Manager not toggling

### ISSUE THREAD

**Issue by HunterOka** [Microsoft/PowerBI-visuals#312]
I am trying to make my custom visual behave as closely as possible as the built in visualizations. When using a Microsoft visualization, when holding Control and clicking a data sections, multiple sections can be selected. When a section is clicked, it toggles between selected and not selected. I would like to emulate this in my visualization using the Selection Manager API.

Right now, when Control-clicking, adding new sections to the selection works. However, when clicking on a selected section attempting to remove it from the selection, instead of toggling, all sections become de-selected. 

How can I make Control-clicking a selected section toggle that section instead of deselecting everything? 

Here are some code snippets. Most of this is based on the "Creating a Custom PowerBI Visualization from Scratch with Sachin Patney" video, with some changes based on reading some of the source code in the repository. 

The visualization is based on stadium seating, with each stadium section being a data point.
```javascript
export interface section
	{
		sectionId: string;
		capacity: number;
		numSeatsSold: number;
		identity: SelectionId;	
	}
```

identity is populated in the converter function:
```javascript
{
     //...
     // dataView.categorical.categories[0] is the array of sections
     // i is the section index
     identity:SelectionIdBuilder.builder().withCategory(dataView.categorical.categories[0],i).createSelectionId();
}
```

In init, create a selectionManager instance

    this.selectionManager = new utility.SelectionManager({hostServices:options.host});

In update
```javascript
//When a section is clicked
section.on('click',function(d){
this.selectionManger.select(d.identity, d3.event.ctrlKey).then(ids=>{
     if(ids.length>0)//If 1 or more sections are selected
     {
          //Modify CSS
     }
     d3.event.stopPropagation();
}

//When empty space is clicked
this.svg.on('click', 
() => this.selectionManager.clear().then(() => /*Modify CSS*/)
```

When a section that is already selected is Control clicked, ids becomes an empty array. Reading the code for SelectionManager, I cannot figure out why. Am I passing in or doing SelectionIds wrong in some way?

**Comment by DenisKudelin** [comment]
Yes, probably it's infrastructure issue. We will add this bug to the backlog.
